Tasks/AzureAppServiceManage/azureappservicemanage.ts

```diff
@@ -104,6 +109,15 @@ async function run() {
                 if (swapWithProduction) {
                     targetSlot = "production";
                 }
+
+                sourceSlot = sourceSlot.toLowerCase();
+                targetSlot = targetSlot.toLowerCase();
+
+                if(sourceSlot == 'production') {
```

Comment:
"production" string is being used at multiple places, define a constant for it.

Reply:
done